helpers/config.py

```diff
@@ -52,18 +52,51 @@ def verify_settings():
     data['general']['parser_opacity'] = get_setting(
         data['general'].get('parser_opacity', 80),
         80,
-        lambda x: (x > 0 and x <= 100)
+        lambda x: (0 < x <= 100)
         )
+    data['general']['window_flush'] = get_setting(
+        data['general'].get('window_flush', True),
+        True
+    )
     data['general']['qt_scale_factor'] = get_setting(
         data['general'].get('qt_scale_factor', 100),
         100,
-        lambda x: (x >= 100 and x <= 300)
+        lambda x: (100 <= x <= 300)
         )
     data['general']['update_check'] = get_setting(
         data['general'].get('update_check', True),
         True
     )
 
+    # sharing
+    data['sharing'] = data.get('sharing', {})
+    data['sharing']['player_name'] = get_setting(
+        data['sharing'].get('player_name', "ConfigureMe"),
+        "ConfigureMe"
+    )
+    data['sharing']['url'] = get_setting(
+        data['sharing'].get('url', "ws://sheeplauncher.net:8424"),
```

Comment:
I don't know if I want to include this default server, or make users put their own in. I run this but I can't guarantee it'll run for the rest of time, obviously. Server code is included, anyone can run their own. I could just leave this as `localhost` but for now this seems workable for convenience?